sunxspex/photon_power_law.py

```diff
@@ -0,0 +1,206 @@
+import functools
+
+import numpy as np
+
+import astropy.units as u
```

Comment:
```suggestion
import astropy.units as u
import numpy as np
```
See section on ordering imports in PEP 8

Reply:
I actually ran `autopep8` on this file multiple times :p

It looks like [this example given in the PEP8 "Imports" section](https://peps.python.org/pep-0008/#imports) shows that the `foo.bar` package should be imported second?
```py
import myclass
import foo.bar.yourclass
```